x/evm/keeper/utils.go

```diff
@@ -77,11 +72,7 @@ func (k Keeper) DeductTxCostsFromUserBalance(
 
 	// deduct the full gas cost from the user balance
 	if err := authante.DeductFees(k.bankKeeper, ctx, signerAcc, fees); err != nil {
-		return nil, stacktrace.Propagate(
-			err,
-			"failed to deduct full gas cost %s from the user %s balance",
-			fees, msgEthTx.From,
-		)
+		return nil, err
```

Comment:
adding addtional info
`failed to deduct full gas cost %s from the user %s balance`

Reply:
kept original error message with `sdkerrors.Wrapf`.